config/routes.rb

```diff
@@ -2,6 +2,7 @@
 
 Rails.application.routes.draw do
   devise_for :accounts
+  resources :accounts, only: %i[index show edit]
```

Comment:
Where is delete action. We should be able to delete account if needed. Maybe not our own account but for this, we can have some kind of protection in the code.